keps/sig-node/24-apparmor/README.md

```diff
@@ -0,0 +1,790 @@
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+  Automated tests will cover the scenarios with and without the changes proposed
+  on this KEP. As defined under [Version Skew Strategy](#version-skew-strategy),
+  we are assuming the cluster may have kubelets with older versions (without
+  this KEP' changes), therefore this will be covered as part of the new tests.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+  
+  The promotion of AppArmor to GA would deprecate the beta annotations as described in the 
+  [Version Skew Strategy](#version-skew-strategy).
+
+### Monitoring requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+  
+  The feature is built into the kubelet and api server components. No metric is
+  planned at this moment. The way to determine usage is by checking whether the
+  pods/containers have a AppArmorProfile set.
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+
+- [ ] Events - Event Reason: 
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
+
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+  N/A
```

Comment:
The most likely error will be manifested by a pod that is unable to run or by rejection rates for pod creation or update.  Finding the kubelet metric for "pod cannot start because of apparmor total" and putting it here will help people trying to debug this in the field.